apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx

```diff
@@ -160,6 +161,7 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
         renderBody: () => (
           <Flex flexDirection="column">
             <TrackPage category={`Set up ${productName}: Step 2 PIN`} flow={analyticsFlowName} />
+            {/* @ts-expect-error weird props issue with React 18 */}
```

Comment:
What is the error? :)

Reply:
Don't remember exactly. Seems like the StepText is not typed correctly with styled-components.

Reply:
Ok, i saw that is what we are doing currentlty so LGTM :) 

But we need to tackle that and fix it when we have a moment